### ISSUE TITLE
Confusion around email attribute type

### ISSUE THREAD

**Issue by rhewitt22** [balderdashy/sails-docs#588]
According to the [docs](https://github.com/balderdashy/sails-docs/blob/master/concepts/ORM/Attributes.md) in order to enforce email validation on an attribute you use:
```
module.exports = {
  attributes: {
    email: {
      type: 'string',
      email: true, <-----
      unique: true,
      required: true
    }
  }
};
```

I tried this with a valid email and got an error.

According to a question on stackoverflow [this may have changed](https://stackoverflow.com/questions/29675698/email-validation-rule-crash-sails-server-mongo-with-sails-js).  The user suggested the following:

```
module.exports = {
  attributes: {
    email: {
      type: 'email', <----
      unique: true,
      required: true
    }
  }
};
```
Which is correct?  If `type: email` is correct I'll submit a PR.

**Comment by sailsbot** [comment]
Thanks for posting, @rhewitt22.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
